lib/wickedbase.pm

```diff
@@ -68,6 +68,25 @@ sub assert_wicked_state {
     assert_script_run("ping -c 4 $args{ping_ip}") if $args{ping_ip};
 }
 
+
+sub reset_wicked {
+    my $self = @_;
+    # Remove any config file and leave the system clean to start tests
+    script_run('find /etc/sysconfig/network/ -name "ifcfg-*" | grep -v "ifcfg-lo" | xargs rm');
+    script_run('rm -f route');
+
+    # Remove any previous manual ip configuration
+    my $iface = iface();
+    script_run("ip a flush dev $iface");
+    script_run('ip r flush all');
+    script_run("ip link set dev $iface down");
+
+    # Restart services
+    script_run('rcwickedd restart');
+    script_run('rcwicked restart');
```

Comment:
this will actually change the interfaces if there is any config file changed. 
Example:
if eth0 is configured according to ifcfg-eth0 and you delete this file, after this command, eth0 will be unconfigured (similar to ifreload).